src/components/testCaseConfiguration/expansion/Expansion.tsx

```diff
@@ -110,7 +110,7 @@ const Expansion = () => {
       .then(() => {
         handleToast("success", "Expansion details Updated Successfully", true);
         // To disable Run test case button, until cqmMeasure is rebuilt
-        setExecutionContextReady(false);
+        // setExecutionContextReady(false);
```

Comment:
If we need to keep this code around (for A/B testing or something), please comment stating why. If not, please remove with it's accompanying code comment.